net/batman-adv-legacy/Makefile

```diff
@@ -112,8 +66,6 @@ endef
 
 define Build/Patch
```

Comment:
We should be able to remove all these explicit Build/* steps as well and use the defaults. Also drop the batctl patches in the *patches* subdir.